src/Components/VolunteerDashboard/VolunteerDashboard.js

```diff
@@ -4,91 +4,140 @@ import ScheduledPickupCards from "./ScheduledPickupCards";
 import AvailablePickupCards from "./AvailablePickupCards";
 import logo from "../../images/logo.png";
 import userImage from "../../images/userImage.jpg";
+import { isEmptyChildren } from "formik";
 
 export default function VolunteerDashboard() {
-  const [pickup, setPickup] = useState([]); //holding pickup data
-  const [volunteer, setVolunteer] = useState(["Volunteer"]); // state holding volunteer name
-  // const [business, setBusiness] = useState([]); // state holding business data: address, etc
+  const id = localStorage.getItem("id");
+  // console.log(id);
 
+  const [available, setAvailable] = useState([]);
+  const [scheduled, setScheduled] = useState([]);
+  const [pickup, setPickup] = useState([]); //holding pickup data
+  const [volunteer, setVolunteer] = useState(["Volunteer"]); // state volunteer name
+  const [restaurants, setRestaurants] = useState([]); // state holding business address
+  // const id = props.match.params.id;
   //axios call for volunteer name
+
+  useEffect(() => {
+    axios
+      .get("http://0bbfee1e.ngrok.io/pickups/")
+      .then(res => {
+        const scheduledPickups = res.data.filter(p => p.volunteer_id === id);
+        setScheduled(scheduledPickups);
+        const avilablePickups = res.data.filter(p => p.volunteer_id === id);
```

Comment:
both pickups are doing the same thing.  filtering when the volunteer_id is the id of the logged in user.  the availablePickups should be when p.volunteer_id == null.  use  = = instead of = = = for soft equality checking.